Flaky test

We have a flaky test.

- [CircleCI](https://app.circleci.com/pipelines/github/honeycombio/hpsf/202) (internal)


```
=== FAIL: pkg/translator TestGenerateConfigForAllComponents/honeycombexporter_all.yaml (0.00s)
    translator_test.go:57: 
                Error Trace:    /home/circleci/project/pkg/translator/translator_test.go:57
                Error:          Not equal: 
                                expected: "AccessKeys:\n    SendKey: key1234\nGeneral:\n    ConfigurationVersion: 2\n    MinRefineryVersion: v2.0\nNetwork:\n    HoneycombAPI: https://alternative.honeycomb.io\n"
                                actual  : "AccessKeys:\n    SendKey: \"\"\nGeneral:\n    ConfigurationVersion: 2\n    MinRefineryVersion: v2.0\nNetwork:\n    HoneycombAPI: https://api.honeycomb.io\n"
                            
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1,3 +1,3 @@
                                 AccessKeys:
                                -    SendKey: key1234
                                +    SendKey: ""
                                 General:
                                @@ -6,3 +6,3 @@
                                 Network:
                                -    HoneycombAPI: https://alternative.honeycomb.io
                                +    HoneycombAPI: https://api.honeycomb.io
                                 
                Test:           TestGenerateConfigForAllComponents/honeycombexporter_all.yaml
    --- FAIL: TestGenerateConfigForAllComponents/honeycombexporter_all.yaml (0.00s)
```

--- a/pkg/translator/testdata/hpsf/deterministicsampler_all.yaml
+++ b/pkg/translator/testdata/hpsf/deterministicsampler_all.yaml
@@ -1,6 +1,6 @@
 components:
   - name: honeycomb_in
-    kind: HoneycombExporter
+    kind: TraceConverter
   - name: honeycomb_out
     kind: HoneycombExporter
   - name: DeterministicSampler_1

--- a/pkg/translator/testdata/hpsf/deterministicsampler_defaults.yaml
+++ b/pkg/translator/testdata/hpsf/deterministicsampler_defaults.yaml
@@ -1,6 +1,6 @@
 components:
   - name: honeycomb_in
-    kind: HoneycombExporter
+    kind: TraceConverter
   - name: honeycomb_out
     kind: HoneycombExporter
   - name: DeterministicSampler_1

--- a/pkg/translator/testdata/hpsf/emathroughput_all.yaml
+++ b/pkg/translator/testdata/hpsf/emathroughput_all.yaml
@@ -1,6 +1,6 @@
 components:
   - name: honeycomb_in
-    kind: HoneycombExporter
+    kind: TraceConverter
   - name: honeycomb_out
     kind: HoneycombExporter
   - name: EMAThroughput_1

--- a/pkg/translator/testdata/hpsf/emathroughput_defaults.yaml
+++ b/pkg/translator/testdata/hpsf/emathroughput_defaults.yaml
@@ -1,6 +1,6 @@
 components:
   - name: honeycomb_in
-    kind: HoneycombExporter
+    kind: TraceConverter
   - name: honeycomb_out
     kind: HoneycombExporter
   - name: EMAThroughput_1

--- a/pkg/translator/testdata/hpsf/honeycombexporter_all.yaml
+++ b/pkg/translator/testdata/hpsf/honeycombexporter_all.yaml
@@ -1,6 +1,6 @@
 components:
   - name: honeycomb_in
-    kind: HoneycombExporter
+    kind: TraceConverter
   - name: honeycomb_out
     kind: HoneycombExporter
     properties:

--- a/pkg/translator/testdata/hpsf/honeycombexporter_defaults.yaml
+++ b/pkg/translator/testdata/hpsf/honeycombexporter_defaults.yaml
@@ -1,6 +1,6 @@
 components:
   - name: honeycomb_in
-    kind: HoneycombExporter
+    kind: TraceConverter
   - name: honeycomb_out
     kind: HoneycombExporter
 connections:

--- a/pkg/translator/translator_test.go
+++ b/pkg/translator/translator_test.go
@@ -28,16 +28,16 @@ func TestGenerateConfigForAllComponents(t *testing.T) {
 	tlater.InstallTemplates(templates)
 	require.Equal(t, templates, tlater.GetTemplates())
 
-	for componentName := range comps {
+	for _, component := range comps {
 		for _, properties := range []string{"all", "defaults"} {
-			testData := fmt.Sprintf("%s_%s.yaml", strings.ToLower(componentName), properties)
+			testData := fmt.Sprintf("%s_%s.yaml", strings.ToLower(component.Kind), properties)
 			t.Run(testData, func(t *testing.T) {
 				// test source config lives in testdata/hpsf
 				b, err := os.ReadFile(path.Join("testdata", "hpsf", testData))
 				require.NoError(t, err)
 				var inputData = string(b)
 
-				for _, template := range comps[componentName].Templates {
+				for _, template := range component.Templates {
 					configType := config.Type(template.Kind)
 					b, err = os.ReadFile(path.Join("testdata", string(configType), testData))
 					require.NoError(t, err)

